packages/grafana-ui/src/components/Table/styles.ts

```diff
@@ -11,14 +11,30 @@ export function useTableStyles(theme: GrafanaTheme2, cellHeightOption: TableCell
   const rowHeight = cellHeight + 2;
   const headerHeight = 28;
 
-  const buildCellContainerStyle = (color?: string, background?: string, overflowOnHover?: boolean) => {
+  const buildCellContainerStyle = (
+    color?: string,
+    background?: string,
+    overflowOnHover?: boolean,
+    asCellText?: boolean
+  ) => {
     return css({
       label: overflowOnHover ? 'cellContainerOverflow' : 'cellContainerNoOverflow',
       padding: `${cellPadding}px`,
       width: '100%',
       // Cell height need to account for row border
       height: `${rowHeight - 1}px`,
-      display: 'flex',
+
+      display: asCellText ? 'block' : 'flex',
```

Comment:
@gtk-grafana This breaks the filter actions that now appear below the row/cell, making the filter action tooltip block out the cell content. 

![Screenshot 2024-02-07 at 15 58 38](https://github.com/grafana/grafana/assets/10999/634c6eb9-d648-4365-9a96-ba103362bf3f)